Update comparison table

TonimatasDEV opened the pull request:
Update some Pelican features and add comparison with Pelican competitors.

Closes #99 

### Frontend:
- Switch from Webpack to Vite. Done. https://github.com/pelican-dev/panel/pull/880
- Rewrite of Client Area in Filament. Done. https://github.com/pelican-dev/panel/pull/416
- Localized Frontend. In Progress. https://github.com/pelican-dev/panel/pull/965

### Server:
- Server Sorting. In Progress? https://github.com/pelican-dev/panel/pull/740

### I can add more things, like:
- https://github.com/pelican-dev/panel/pull/469
- https://github.com/pelican-dev/panel/pull/1002 and https://github.com/pelican-dev/panel/pull/892
- Node: Self Update. https://github.com/pelican-dev/wings/pull/51
- ? https://github.com/pelican-dev/panel/pull/896 
- ? https://github.com/pelican-dev/wings/pull/31

TonimatasDEV commented:
The comparison with Pelican's competitors should look like this:
![{2126093D-BACE-4493-A9E5-796A0173B278}](https://github.com/user-attachments/assets/92f94dac-face-4773-9010-3bf3623bec6e)

I need to review the table information to make sure it's accurate. I can add or remove items if you would like.

TonimatasDEV commented:
> Update some Pelican features and add comparison with Pelican competitors (#99).
> 
> ### Frontend:
> * Switch from Webpack to Vite. Done. [Remove old client area and switch to vite panel#880](https://github.com/pelican-dev/panel/pull/880)
> * Rewrite of Client Area in Filament. Done. [feat: Client UI translate to Filament (from React) panel#416](https://github.com/pelican-dev/panel/pull/416)
> * Localized Frontend. In Progress. [Admin Area Translations panel#965](https://github.com/pelican-dev/panel/pull/965)
> 
> ### Server:
> * Server Sorting. In Progress? [Add (global) sorting to server list panel#740](https://github.com/pelican-dev/panel/pull/740)
> 
> ### I can add more things, like:
> * [Health page panel#469](https://github.com/pelican-dev/panel/pull/469)
> * [Alert banner panel#1002](https://github.com/pelican-dev/panel/pull/1002) and [Alert banners panel#892](https://github.com/pelican-dev/panel/pull/892)
> * Node: Self Update. [add selfupdate wings#51](https://github.com/pelican-dev/wings/pull/51)
> * ? [PostgreSQL Support panel#896](https://github.com/pelican-dev/panel/pull/896)
> * ? [Wings: Fix stoplogic wings#31](https://github.com/pelican-dev/wings/pull/31)

Boy132 commented:
> The comparison with Pelican's competitors should look like this: ![{2126093D-BACE-4493-A9E5-796A0173B278}](https://private-user-images.githubusercontent.com/79017964/418376067-92f94dac-face-4773-9010-3bf3623bec6e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA5OTEzNDgsIm5iZiI6MTc0MDk5MTA0OCwicGF0aCI6Ii83OTAxNzk2NC80MTgzNzYwNjctOTJmOTRkYWMtZmFjZS00NzczLTkwMTAtM2JmMzYyM2JlYzZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzAzVDA4MzcyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2NGU0ODg3YjI3MDFiZTJkYzJiNzVlMTE4MGMxNDQ5OTQwNmRiZjRlZmI3ZjBmNDkyOTYzMGY2ZDQ4ODMxYjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zjxi8qbQoaWH1IvArLiOkWif0VAwrRhe3da2RYZh0nY)
> 
> I need to review the table information to make sure it's accurate. I can add or remove items if you would like.

Looks good for starters. Can you also add AMP & TCAdmin?

Also, I'm all for doing an "honest" comparison. By that I mean list features that other panels have but Pelican don't. Thoughts @pelican-dev/panel ?

TonimatasDEV commented:
> Also, I'm all for doing an "honest" comparison. By that I mean list features that other panels have but Pelican don't. Thoughts @pelican-dev/panel ?

I added more features that Pelican panel does not have and updated the table.

TonimatasDEV commented:
Private networks and ipv6 are good features to this table?

rmartinoscar commented on docs/comparison.mdx:
```diff
@@ -83,3 +83,23 @@ Pelican is a fork of [Pterodactyl](https://pterodactyl.io). But over time Pelica
+|                          | Pelican | Pterodactyl | PufferPanel | Crafty Controller     | Multicraft     | TCAdmin | AMP            |
+|--------------------------|---------|-------------|-------------|-----------------------|----------------|---------|----------------|
+| File manager             | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
+| FTP file access          | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
```
```suggestion
| FTP file access          | SFTP       | SFTP           | SFTP           | SFTP                     | FTP              | FTP       | SFTP              |
```

rmartinoscar replied:
Or make a new line

rmartinoscar commented on docs/comparison.mdx:
```diff
@@ -83,3 +83,23 @@ Pelican is a fork of [Pterodactyl](https://pterodactyl.io). But over time Pelica
+| FTP file access          | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
+| Scheduled Tasks          | ✅       | ✅           | ❌           | ✅                     | ✅              | ✅       | ✅              |
+| Free and Open Source     | ✅       | ✅           | ✅           | ✅                     | ❌              | ❌       | ❌              |
+| Backups system           | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
```
```suggestion
| Backups system           | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
```
We should show that we have s3 support aswell as local ones

rmartinoscar commented on docs/comparison.mdx:
```diff
@@ -83,3 +83,23 @@ Pelican is a fork of [Pterodactyl](https://pterodactyl.io). But over time Pelica
+| Backups system           | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
+| Multi-language           | ✅       | ✅           | ✅           | ✅                     | ✅              | ✅       | ✅              |
+| Database Management      | ✅       | ✅           | ❌           | ❌                     | ✅              | ✅       | ✅              |
+| Captcha                  | ✅       | ✅           | ❌           | ❌                     | ❌              | ❌       | ❌              |
```
We have turnstile which is backwards compatible with Captcha

TonimatasDEV commented:
I added the PHP version 8.4. It should work: https://github.com/pelican-dev/panel/pull/858
